include/circt/Dialect/Calyx/CalyxPrimitives.td

```diff
@@ -332,10 +332,17 @@ def AndLibOp  : CombinationalArithBinaryLibraryOp<"and"> {}
 def OrLibOp   : CombinationalArithBinaryLibraryOp<"or"> {}
 def XorLibOp  : CombinationalArithBinaryLibraryOp<"xor"> {}
 
-class ArithBinaryFloatingPointLibraryOp<string mnemonic> : ArithBinaryLibraryOp<mnemonic, [
-  SameTypeConstraint<"left", "out">]> {}
+def FloatingPoint : NativeOpTrait<"FloatingPoint"> {
+  let cppNamespace = "::circt::calyx";
+}
+
+class ArithBinaryFloatingPointLibraryOp<string mnemonic> :
+  ArithBinaryLibraryOp<mnemonic, "", [
+    FloatingPoint,
+    SameTypeConstraint<"left", "out">
+  ]> {}
 
-def AddFNOp : ArithBinaryFloatingPointLibraryOp<"addFN"> {
+def AddFNOp : ArithBinaryFloatingPointLibraryOp<"ieee754.add"> {
```

Comment:
I like this better. IMO, a better name for this might be: `AddFOpIeee754` and similar for below. I imagine future ops would follow similar patterns, e.g., `AddFOpBf16`.